examples/v1alpha1/schema_write_example.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	pb "github.com/authzed/authzed-go/proto/authzed/api/v1alpha1"
+	authzed "github.com/authzed/authzed-go/v1alpha1"
+	"github.com/authzed/grpcutil"
+)
+
+const schema = `definition blog/user {}
```

Comment:
Change the prefix to be a variable and then put a comment indicating that it needs to be replaced

Reply:
this isn't new code it's from the docs